nitrocli/README.md

```diff
@@ -125,6 +124,16 @@ The [Nitrokey UG][nitrokey-ug] has generously provided the necessary
 hardware for developing and testing the program.
 
 
+License
+-------
+**nitrocli** is made available under the terms of the
+[GPLv3](gplv3-tldr).
+
+See the [LICENSE] file that accompanies this distribution for the full
```

Comment:
I think this should be `[LICENSE](LICENSE)`.

Reply:
Good catch, thanks!